QA.md

```diff
@@ -140,22 +169,22 @@ You can connect your wallet to the testnet (https://node1.foxtrot.testnet.hathor
 1. **Register tokens with same name**
     1. Unregister TST token but copy its configuration string and save it.
     1. Click on 'Custom tokens' and then 'Create a new token'.
-    1. Type 'Test token' in the token name, 'Test' in the symbol and 200 in the amount.
+    1. Type 'Test token' in the token name, 'TST' in the symbol and 200 in the amount.
     1. Now click on 'Custom tokens', then 'Register a token'.
     1. Type 'abc'. It should show an error of invalid configuration string.
     1. Type the configuration string saved in the first step. It must appear an error that you already have a token with this name.
 
 1. **Try to spend same output**
     1. Click on 'Wallet' in the navigation bar, select the HTR token and copy the address (and save it).
-    1. Click on the first transaction (the one with amount 0.00).
+    1. Click on the first transaction with amount 0.00.
     1. Copy the first input from HTR and from your address (copy the transaction ID and the index).
     1. Click on 'Send tokens'.
     1. Paste the address and 1.00 as the amount. Deselect 'Choose inputs automatically' and paste the transaction ID and index copied before.
     1. Must see an error that the output is already spent.
 
 1. **Create NFT**
     1. Click on 'Custom tokens', then 'Create an NFT'.
-    1. Fill NFT data as 'test data', name and symbol as 'NFT Test' and 'NFTT', and amount as 100. Check only the melt authority checkbox.
+    1. Fill NFT data as `ipfs://test`, name and symbol as 'NFT Test' and 'NFTT', and amount as 100. Check only the melt authority checkbox.
```

Comment:
Oh yea, that would work too

But I think we can remove the ipfs:// from the next step that uses it

Reply:
My intent here was just to keep the data consistent with the validation we make a few steps after.
Keeping both `ipfs://test` or `test data` would suffice.